src/overlays/actors/ovl_En_Heishi2/z_en_heishi2.h

```diff
@@ -3,12 +3,53 @@
 
 #include <ultra64.h>
 #include <global.h>
+#include <vt.h>
+#include "overlays/actors/ovl_Bg_Gate_Shutter/z_bg_gate_shutter.h"
+#include "overlays/actors/ovl_En_Bom/z_en_bom.h"
+#include "overlays/actors/ovl_Bg_Spot15_Saku/z_bg_spot15_saku.h"
 
 struct EnHeishi2;
 
 typedef struct EnHeishi2 {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x298];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ Vec3s limbDrawTable[17];
+    /* 0x01F6 */ Vec3s transitionDrawTable[17];
+    /* 0x025C */ ActorFunc actionFunc;
+    /* 0x0260 */ Vec3s unk_260;
```

Comment:
with the newly merged actor changes, we are no longer using ActorFunc for this. Every actor has their own typedef. Check out other actors for an example